src/components/headers_overlay/headers_overlay.ts

```diff
@@ -489,8 +491,20 @@ export class ColResizer extends AbstractResizer {
     });
   }
 
-  getUnhideButtonStyle(hiddenIndex: HeaderIndex): string {
-    return cssPropertiesToCss({ left: this._getDimensionsInViewport(hiddenIndex).start + "px" });
+  getUnhideButtonStyle(hiddenIndexes: HeaderIndex[]): string {
+    const lastIndex =
+      this.env.model.getters.getNumberCols(this.env.model.getters.getActiveSheetId()) - 1;
+
+    let leftOffset = this._getDimensionsInViewport(hiddenIndexes[0]).start;
+    if (hiddenIndexes.includes(0)) {
+      leftOffset += 10;
+    } else if (hiddenIndexes.includes(lastIndex)) {
+      leftOffset -= 10;
+    }
```

Comment:
Don't put arbitraty offset with no link to anything. The day we change exact sizes in the CSS this will become wrong. furthermore ATM the arrows are displaced with the transform CSS properties. Which is wrong for the extremities of the sheet. What your offset does in essence is to leave the wrong transform, but compensate the wrong transform so they become right.

We should either:
a) overwriting tranform CSS for the first/last col/row to it's correct
b) (rpobably better) leave the transform be, and apply CSS hidden to the arrows on the first/last col instead of the t-if. So the transform -50% will always be correct, since it takes both arrows into account. (and this can be done with a t-att-aclass in the tempalte directly)

Reply:
Changes made as suggested. Thanks!